developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,116 @@
+## Signature scheme identifiers
 
-## Account resources
+Generating the authentication key for an account requires that you provide one of the below 1-byte signature scheme identifiers for this account, i.e., whether the account is a single signature or a multisig account:
+
+- **1-byte single-signature scheme identifier**: `0x00`.
+- **1-byte multisig scheme identifier**: `0x01`. Make sure to also provide the value of `K` to generate the K-of-N multisig authentication key.
+
+## Single signer authentication
+
+To generate an authentication key and the account address for a single signature account:
+
+1. **Generate a key-pair**: Generate a fresh key-pair (`privkey_A`, `pubkey_A`). The Aptos Blockchain uses the PureEdDSA scheme over the Ed25519 curve, as defined in RFC 8032.
+2. **Derive a 32-byte authentication key**: Derive a 32-byte authentication key from the `pubkey_A`:
+     ```
+     auth_key = sha3-256(pubkey_A | 0x00)
+     ```
+     where `|` denotes concatenation. The `0x00` is the 1-byte single-signature scheme identifier. 
+3. Use this initial authentication key as the permanent account address.
+
+## Multisigner authentication
+
+With K-of-N multisig authentication, there are a total of N signers for the account, and at least K of those N signatures must be used to authenticate a transaction.
+
+To generate a K-of-N multisig account's authentication key and the account address:
+
+1. **Generate key-pairs**: Generate `N` ed25519 public keys `p_1`, ..., `p_n`.
+2. Decide on the value of `K`, the threshold number of signatures needed for authenticating the transaction.
+3. **Derive a 32-byte authentication key**: Compute the authentication key as described below:
+    ```
+    auth_key = sha3-256(p_1 | . . . | p_n | K | 0x01)
+    ```
+    The `0x01` is the 1-byte multisig scheme identifier.
+4. Use this initial authentication key as the permanent account address.
+
+:::tip Accounts on Aptos Testnet
+In order to create accounts, the Aptos testnet requires the account's public key and an amount of `Coin<TestCoin>` to add to that account, resulting in the creation of a new account with those two resources.
+:::
+
+## Access control with signer
+
+The sender of a transaction is represented by a signer. When a signer is specified in a Move module function in a transaction, then the Move VM in the Aptos blockchain will assert that the signer of the transaction (that triggered that function) is the owner of the account. See the below Move example code with `signer` in the `initialize` and `withdraw` functions. When a `signer` is not specified in a function, for example, the below `deposit` function, then no access controls exist for this function:
```

Comment:
why is this "when a signer is specified"

The Aptos Move VM adapter translates the identity of the account that signed the transaction into a signer in a Move module entry point...

Reply:
This is referring to the move function I believe. So perhaps to be clearer it should be "when a function in a Move module takes `signer` as its first argument".